src/overlays/actors/ovl_En_Fish/z_en_fish.c

```diff
@@ -42,87 +81,875 @@ const ActorInit En_Fish_InitVars = {
+    Collider_SetJntSph(globalCtx, &this->collider, &this->actor, &sJntSphInit, this->colliderElements);
+
+    this->actor.colChkInfo.mass = this->unk_25C * 30.0f;
+    this->unk_244 = (u32)Rand_Next() >> 0x10;
+    this->unk_246 = (u32)Rand_Next() >> 0x10;
+
+    if (sp36 == ENFISH_0) {
+        this->actor.flags |= 0x10;
+        ActorShape_Init(&this->actor.shape, 0.0f, func_800B3FC0, 8.0f);
+        func_8091E810(this);
+    } else if (sp36 == ENFISH_1) {
+        func_8091F344(this);
+    } else {
+        func_8091E070(this);
+    }
+}
+
+void EnFish_Destroy(Actor* thisx, GlobalContext* globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
+    EnFish* this = THIS;
+
+    Collider_DestroyJntSph(globalCtx, &this->collider);
+}
+
+void func_8091DD48(EnFish* this) {
+    this->actor.shape.yOffset += (Math_SinS(this->unk_246) * 5.0f) + (Math_SinS(this->unk_244) * 10.0f);
+
+    if (this->actor.shape.yOffset < -200.0f) {
+        this->actor.shape.yOffset = -200.0f;
+    } else if (this->actor.shape.yOffset > 200.0f) {
+        this->actor.shape.yOffset = 200.0f;
+    }
+}
+
+s32 func_8091DDF4(EnFish* this, GlobalContext* globalCtx) {
+    s32 pad;
+    Player* player = GET_PLAYER(globalCtx);
+    Vec3f sp1C;
+
+    if (this->actor.xzDistToPlayer < 50.0f) {
+        sp1C.x = (Math_SinS(BINANG_ROT180(this->actor.yawTowardsPlayer)) * 30.0f) + player->actor.world.pos.x;
+        sp1C.y = player->actor.world.pos.y;
+        sp1C.z = (Math_CosS(BINANG_ROT180(this->actor.yawTowardsPlayer)) * 30.0f) + player->actor.world.pos.z;
+
+        if (func_8091D630(&sp1C, &this->actor.world.pos) <= 900.0f) {
+            return true;
+        }
+    }
+    return false;
+}
+
+void func_8091DEE4(EnFish* this) {
+    static s16 D_8091FACC[] = { 85, 60, 45 };
+
+    this->actor.gravity = 0.0f;
+    this->actor.minVelocityY = 0.0f;
+    this->unk_240 = Rand_S16Offset(5, D_8091FACC[this->unk_278]);
+    this->unk_248 = 0;
+    this->unk_26E = 400;
+    this->unk_272 = 400;
+    this->unk_268 = 0;
+    this->unk_26C = 0;
+    func_8091D660(this);
+    this->unk_24C = 0.0f;
+    this->unkFunc = func_8091DF68;
+}
+
+void func_8091DF68(Actor* thisx, GlobalContext* globalCtx) {
+    EnFish* this = THIS;
+
+    func_8091DD48(this);
+    Math_SmoothStepToF(&thisx->speedXZ, 0.0f, 0.05f, 0.3f, 0.0f);
+
+    if ((thisx->speedXZ * 1.4f) + 0.8f > 2.0f) {
+        this->skelAnime.playSpeed = 2.0f;
+    } else {
+        this->skelAnime.playSpeed = (thisx->speedXZ * 1.4f) + 0.8f;
+    }
+
+    this->unk_270 >>= 1;
+
+    if (this->unk_240 <= 0) {
+        func_8091E070(this);
+    } else if (thisx == thisx->child) {
+        func_8091E5EC(this);
+    } else if ((thisx->xzDistToPlayer < 60.0f) || (this->unk_276 > 0)) {
+        func_8091E2E0(this);
+    }
+}
+
+void func_8091E070(EnFish* this) {
+    static s16 D_8091FAD4[] = {
+        20,
+        30,
+        40,
+    };
```

Comment:
```suggestion
    static s16 D_8091FAD4[] = {
        20,
        30,
        40
    };
```
You can remove the comma when they format into a vertical column like this